stages/org.osbuild.timezone

```diff
@@ -0,0 +1,25 @@
+#!/usr/bin/python3
+
+import json
+import subprocess
+import sys
+
+
+def main(tree, options):
+    zone = options["zone"]
+
+    # We need to remove the /etc/localtime symlink first, because it is created while we install RPM packages.
+    # systemd-firstboot expects that if /etc/localtime exists it is a user-defined value and does not change it, but
+    # the assumption is wrong, because it contains a default value from RPM package.
+    # This is (hopefully) a temporary workaround.
+    subprocess.run(["rm", f"{tree}/etc/localtime"], check=True)
```

Comment:
How about using `os.remove()` in a try/except and catch the case when the file is not there. That way we don't need to update this if the symlink is no longer shipped?

Reply:
Right, we were discussing this above. I'm fine with both, having a comment explaining why it fails if it starts failing (which @msehnout did after my comment) or disabling the check. `os.remove()` is indeed preferable to spawning `rm`.

Reply:
updated